pkg/ipam/crd_test.go

```diff
@@ -59,6 +60,7 @@ func TestIPNotAvailableInPoolError(t *testing.T) {
 }
 
 var testConfigurationCRD = &option.DaemonConfig{
+	ConfigPatchMutex:        new(lock.RWMutex),
```

Comment:
Makes my eyes bleed that every test now needs to do this. OTOH, maybe this encourages moving away from the global DaemonConfig.. :D